### PR TITLE
Fix #3697 - Selenium 3.0.1 problem - wrong jvm params

### LINKED ISSUE #3697: Selenium 3.0.1 problem - wrong jvm params
Bug report
---

- Node Version: `v6.9.1`
- Protractor Version: `4.0.10`
- Operating System and Version `Windows`
- Your protractor configuration file

IE config
```javascript
/**
 * Internet Explorer Protractor
 *
 * @type {Object}
 */
var seleniumPath = './node_modules/protractor/node_modules/webdriver-manager/selenium/';

exports.config = {
	allScriptsTimeout: 11000,

	seleniumArgs: ['-Dwebdriver.ie.driver='+ seleniumPath +'IEDriverServer_Win32_2.53.1.exe'],

	capabilities: {
		'browserName': 'internet explorer',
        'platform': 'ANY',
        'version': '11'
	},

	baseUrl: 'http://localhost:9000/',

	framework: 'jasmine',

	jasmineNodeOpts: {
		defaultTimeoutInterval: 30000
	}
};
```

Firefox:
```javascript
/**
 * Firefox Protractor
 *
 * @type {Object}
 */
var seleniumPath = './node_modules/protractor/node_modules/webdriver-manager/selenium/';

exports.config = {
	allScriptsTimeout: 11000,

	seleniumArgs: ['-Dwebdriver.gecko.driver='+ seleniumPath +'geckodriver-v0.11.1.exe'],
    seleniumServerJar: '../../' + seleniumPath + 'selenium-server-standalone-3.0.1.jar',

	capabilities: {
		browserName: 'firefox',
	},

	baseUrl: 'http://localhost:9000/',

	framework: 'jasmine',

	jasmineNodeOpts: {
		defaultTimeoutInterval: 30000
	}
};
```

- Output from running the test

When I run test I get error:
```bash
Running "protractor:app" (protractor) task
 70% 1/1 build modules[17:58:43] I/local - Starting selenium standalone server...
[17:58:43] I/launcher - Running 1 instances of WebDriver
 70% 5/5 build modules[17:58:43] E/launcher - Server terminated early with status 1
[17:58:43] E/launcher - Error: Server terminated early with status 1
    at Error (native)
    at C:\dev\flicaui\node_modules\protractor\node_modules\selenium-webdriver\remote\index.js:242:20
    at ManagedPromise.invokeCallback_ (C:\dev\flicaui\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1379:14)
    at TaskQueue.execute_ (C:\dev\flicaui\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2913:14)
    at TaskQueue.executeNext_ (C:\dev\flicaui\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2896:21)
    at asyncRun (C:\dev\flicaui\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2775:27)
    at C:\dev\flicaui\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:639:7
    at process._tickCallback (internal/process/next_tick.js:103:7)
[17:58:43] E/launcher - Process exited with error code 199
>>
Warning: Tests failed, protractor exited with code: 199 Use --force to continue.

Aborted due to warnings.

```

- Steps to reproduce the bug
  just use my config files. 

- Additional information. 
I found the bug, so I thought the problem is in Selenium Driver. I created the pull request but the problem is in protractor code. Please look to the fix in selenium repo: https://github.com/SeleniumHQ/selenium/pull/3052 and read comments.

### PR DESCRIPTION
**Pull request by sptq**

### CODE REVIEW

**Comment by sptq** [comment]
I signed it!

File: lib/config.ts [hunk 1]
```diff
@@ -46,6 +46,13 @@ export interface Config {
      * seleniumArgs: ['-browserTimeout=60']
      */
     args?: any;
+
+    /**
+     * Additional command line jvm options to pass to selenium. For example,
+     * if you need to change the browser driver, use
+     * jvmArgs: ['-Dwebdriver.ie.driver=IEDriverServer_Win32_2.53.1.exe']
+     */
+    jvmArgs?: any;
```
Comment (cnishina) [inline lib/config.ts]: This should be `jvmArgs?: string[];`

File: lib/config.ts [hunk 2]
```diff
@@ -580,6 +587,7 @@ export interface Config {
   ng12Hybrid?: boolean;
 
   seleniumArgs?: Array<any>;
+  jvmArgs?: Array<any>;
```
Comment (cnishina) [inline lib/config.ts]: This should be `jvmArgs?: string[];`

**Comment by cnishina** [comment]
Please follow [CONTRIBUTING.md commit message](https://github.com/angular/protractor/blob/master/CONTRIBUTING.md#commit-messages) and squash your commit. Thanks.

**Comment by cnishina** [comment]
Made changes and closed with f3938f9fcd1b91272f2f11d4d39e458576fb75b0

**Pull request closed without merging**